sqlmesh/dbt/builtin.py

```diff
@@ -144,9 +145,9 @@ def no_log(msg: str, info: bool = False) -> str:
     return ""
 
 
-def generate_var(variables: t.Dict[str, t.Any]) -> t.Callable:
+def generate_var(variables: t.Dict[str, t.Any], jinja_env: Environment) -> t.Callable:
     def var(name: str, default: t.Optional[str] = None) -> str:
-        return variables.get(name, default)
+        return jinja_env.from_string(str(variables.get(name, default))).render()
```

Comment:
I don't think we can render vars this late. They may be using env_var and need the local environment. Can we render during loading?

Reply:
Ah, excellent point! Where do you think would be a good place to render these variables?

Reply:
How about here: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/dbt/loader.py#L97?

When building the jinja environment, make sure macros in the same package can be accessed without pre-pending the package name. `package.macro_name` should also work as `macro_name`.